pages/common/fold.md

```diff
@@ -0,0 +1,15 @@
+# fold
+
+> Wraps each line in an input file to fit a specified width.
+
+- Wrap each line to default width (80):
+
+`fold {{file}}`
+
+- Wrap each line to width '30':
+
+`fold -w30 {{file}}`
+
+- Wrap each line to width '5' and break at spaces (If word length > 5, it is wrapped, else only one word is kept per line ):
```

Comment:
I don't understand the note in parenthesis. Care to expand a bit?

minor notpick: the sentence before the parenthesis wasn't terminated, so the "if" should be in lowercase.